pkg/storageops/aws/aws_test.go

```diff
@@ -1,4 +1,4 @@
-package aws_test
+package aws
```

Comment:
having a separate package name `aws_test` allows to test the package as if the tester was an external package. If the test package name is the same as the package being tested, the test package can also use methods and variables not exposed to the eventual user.

Reply:
That is a good thing. Unit tests should access internal structures and functions to be tested. You are describing functional tests.

Reply:
Discussed offline. The goal is to also test internal functions and hence using the same package name.